tensorboard/webapp/metrics/store/metrics_store_internal_utils_test.ts

```diff
@@ -1191,4 +1196,32 @@ describe('metrics store utils', () => {
       });
     });
   });
+
+  describe('cardSelectionStateToBoolean', () => {
+    it('returns true when selection state is ENABLED', () => {
```

Comment:
To really make me confident that this statement is true, you could consider adding a second check:
```
      expect(
        cardSelectionStateToBoolean(CardSelectionState.ENABLED, true)
      ).toBeTrue();
```